.github/workflows/release_sdk.yml

```diff
@@ -37,21 +37,7 @@ jobs:
 
       - name: Setup Python Environment
         run: |
-          make sdk-setup
-          source sdk/.venv/bin/activate
+          make sdk/setup
 
       - name: SDK Release
-        run: make sdk-release
-
-  cleanup:
```

Comment:
As above but less relevant tbh

Reply:
Yeah, here there is no container creation so we're fine without the cleanup